packages/react/src/__tests__/ReactTypeScriptClass-test.ts

```diff
@@ -702,11 +702,4 @@ describe('ReactTypeScriptClass', function() {
     ]);
     expect(ref.current.refs.inner.getName()).toBe('foo');
   });
-
-  it('supports drilling through to the DOM using findDOMNode', function() {
-    const ref = React.createRef();
-    test(React.createElement(Inner, {name: 'foo', ref: ref}), 'DIV', 'foo');
-    const node = ReactDOM.findDOMNode(ref.current);
-    expect(node).toBe(container.firstChild);
-  });
```

Comment:
I had trouble accessing internals from these tests so I just deleted the test. I can undo if there is pushback here but I'd argue that this no longer needs assertions since the API will be removed from public builds